### ISSUE TITLE
Card io scanning wrong Card Number and Expiry date 

### ISSUE THREAD

**Issue by mukeshgupta550** [card-io/card.io-iOS-SDK#236]
Can anyone help or suggest who faced similar kind of issue mentioned below on iOS platform latest card.io version 5.4.1.

This generally happens when you have placed multiple (at least two) cards at your desk and moving the camera focus from one to another card.
i.e If we try to scan Card A and Immediately(after 5-10 second) try to scan other card (Card B) then Card A's Number or expiry date is being scanned which is invalid for Card B.

So as per my analysis, this is card.io limitation. If anyone is facing same issue kindly share your's thought.

**Comment by MiralDesai** [comment]
Bit worrying that nobody has looked at it, but we are facing similar issues. Seems like it is a limitation of card.io. I'm now looking at alternatives. Just posted my own issue and noticed this

**Comment by pzmudzinski** [comment]
@MiralDesai have you found any reliable alternative?

**Comment by MiralDesai** [comment]
@pzmudzinski  Unfortunately not. Seems like CardIO have a bit of a monopoly. I'm no longer working on iOS so not up to date on possible alternatives, hopefully you find something.

**Comment by pzmudzinski** [comment]
Yeah, haven't found anything good, and this SDK seems to be dead, especially when nobody responds to any issues.
